### ISSUE TITLE
Assertion failed on parsing malformed Beta file (main/entry.c:1108)

### ISSUE THREAD

**Issue by kcwu** [universal-ctags/ctags#537]
How to reproduce
```
$ echo -e '--0\t0:--' > min.bet
$ ctags min.bet
ctags: main/entry.c:1108: void initTagEntryFull(tagEntryInfo *const, const char *const, unsigned long, const char *, fpos_t, const char *): Assertion `name == NULL || strchr (name, '\t') == NULL' failed.
ctags: main/entry.c:1108: parsing min.bet:1 as BETA
Aborted
```

this is found by afl-fuzz

**Comment by masatake** [comment]
We have to find a NORMAL test case first.

**Comment by masatake** [comment]
Solved by #582. Thank you.
